### PR TITLE
Push images to the public registry

### PR DESCRIPTION
**Pull request by gavin-stackrox**
Now that StackRox is open source 🎉, there are a number of other repos and processes that depend on images being found under quay.io/stackrox-io. This PR pushes scanner images there also.

### CODE REVIEW

File: scripts/ci/lib.sh [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+    # Push to quay.io/stackrox-io
+    docker login -u "$QUAY_STACKROX_IO_RW_USERNAME" --password-stdin <<<"$QUAY_STACKROX_IO_RW_PASSWORD" quay.io
+    _tag_image_set "quay.io/stackrox-io" "$tag"
+    _push_image_set "quay.io/stackrox-io" "$tag"
+
+    if [[ -n "$ci_tag" ]]; then
+        require_environment "STACKROX_IO_PUSH_USERNAME"
+        require_environment "STACKROX_IO_PUSH_PASSWORD"
+        docker login -u "$STACKROX_IO_PUSH_USERNAME" --password-stdin <<<"$STACKROX_IO_PUSH_PASSWORD" stackrox.io
```
Comment (RTann) [inline scripts/ci/lib.sh:69]: Will this log be exposed in CI? Same for the other `docker login ... --password-stdin...`

**Review by RTann — COMMENTED** [review]
I see 
```
INFO: Fri Apr  1 22:21:14 UTC 2022: Pushing images
Error response from daemon: No such image: scanner:2.23.0-28-g405d65b7ef
```

I'm guessing this is related?

File: scripts/ci/lib.sh (hunk 1 above)
Comment (gavin-stackrox) [reply scripts/ci/lib.sh:69]: Yeah. `--password-stdin` is more secure in theory.
 https://docs.docker.com/engine/reference/commandline/login/#provide-a-password-using-stdin

**Review by RTann — APPROVED** [review]
Merge with CI, thanks!

**Comment by gavin-stackrox** [comment]
> I see
> 
> ```
> INFO: Fri Apr  1 22:21:14 UTC 2022: Pushing images
> Error response from daemon: No such image: scanner:2.23.0-28-g405d65b7ef
> ```
> 
> I'm guessing this is related?

Yeah I clicked review a little too early :( Looking greener now though!

**Pull request merged**